cloud: data service supports basic authentication (#14449)
pingcap/docs: tidb-cloud/data-service-manage-endpoint.md

    >
    > Alternatively, you can also click the endpoint name to view its details and click **...** > **Code Example** in the upper-right corner.

4. In the dialog box, select the environment and authentication method that you want to use to call the endpoint, and then copy the code example.

    > **Note:**
    >
    > - The code examples are generated based on the properties and parameters of the endpoint.
    > - Currently, TiDB Cloud Data Service only provides the curl code example.

    - Environment: choose **Test** or **Online** depending on your need. The **Online** environment is available only after you deploy the endpoint.
    - Authentication method: choose **Basic Auth** or **Digest Auth**.

>         - **Basic Auth** transmits your API key as plain text.

Author: **Basic Auth** transmits your API key as plain text. 是 as plain text 还是 as based64 encoded text 呢？

>         - **Basic Authentication** transmits your API key as plain text.

Author: Basic Authentication transmits your API key as plain text. 是 as plain text 还是 as based64 encoded text 呢？

>         - **Basic Authentication** transmits your API key as plain text.
>         - **Digest Authentication** transmits your API key in an encrypted form so it is more secure.

Author: ```suggestion
        - **Basic Authentication** transmits your API key as based64 encoded text.
        - **Digest Authentication** transmits your API key in an encrypted form so it is more secure.
```

>         - **Digest Authentication** transmits your API key in an encrypted form so it is more secure.
> 
>       Compared with **Basic Authentication**, the curl code of **Digest Authentication** includes an additional `--digest`option.

Reviewer: ```suggestion
        - **Digest Authentication** transmits your API key in an encrypted form, which is more secure.

      Compared with **Basic Authentication**, the curl code of **Digest Authentication** includes an additional `--digest` option.
```
